### ISSUE TITLE
Maintainers welcome :)

### ISSUE THREAD

**Issue by jackfranklin** [jackfranklin/so-fetch#22]
@JohnPhoto thank you so much for your PRs - I'm sorry that I've not had time to look at these.

As you can probably tell I don't have a lot of time to dedicate to this library - it's not one I use much anymore - but I'm very happy to publish new versions with new features. I was wondering if you (or anyone else reading this!) would like to become a contributor on the repo and push this library forwards?

**Comment by JohnLindahlTech** [comment]
Yes I would like to help out on this, and will be willing to become a contributor.
